trillian_log_api.proto

```diff
@@ -14,75 +14,106 @@
 
 syntax = "proto3";
 
+package trillian;
+
+option go_package = "github.com/google/trillian";
 option java_multiple_files = true;
-option java_package = "com.google.trillian.proto";
 option java_outer_classname = "TrillianLogApiProto";
-option go_package = "github.com/google/trillian";
-
-package trillian;
+option java_package = "com.google.trillian.proto";
 
-import "trillian.proto";
-import "google/rpc/status.proto";
 import "google/api/annotations.proto";
 import "google/protobuf/timestamp.proto";
+import "google/rpc/status.proto";
+import "trillian.proto";
+
+// Provides access to a Verifiable Log as defined in the [Verifiable Data
```

Comment:
Can you tidy this comment while you're there. It says "provides access" and "provides direct access". Could be shortened.

Reply:
Done.